assets/javascripts/overview.js

```diff
@@ -301,17 +303,32 @@ function addComments(form) {
     controls.style.display = 'inline';
     window.addCommentsModal.hide();
   };
-  fetchWithCSRF(form.action, {method: 'POST', body: new FormData(form)})
+
+  let infoText =
+    'The comments have been created. <a href="javascript: location.reload()">Reload</a> the page to show changes.';
+  let errText = 'The comments could not be added:';
+  if (actionBtn === 'restartAndCommentJobs') {
+    infoText = '<a href="javascript: location.reload()">Reload</a> the page to show restarted jobs.';
+    errText = 'Failed to restart jobs: ';
+  }
+  fetchWithCSRF(reqUrl, {method: 'POST', body: new FormData(form)})
     .then(response => {
       if (!response.ok) throw `Server returned ${response.status}: ${response.statusText}`;
-      addFlash(
-        'info',
-        'The comments have been created. <a href="javascript: location.reload()">Reload</a> the page to show changes.'
-      );
+      addFlash('info', infoText);
       done();
+      return response.json();
+    })
+    .then(resData => {
+      if (resData.errors.length > 0) {
+        for (let i in resData.errors) {
+          addFlash('warning', 'Warning: Errors found in Response:\n' + resData.errors[i].trim());
+        }
+        done();
+        console.log(resData);
+      }
```

Comment:
As discussed restartJob in openqa.js could be re-used if you adapt it to taking an array of multiple jobs. And maybe just reload the page in case of multiple jobs.
Might also be good to restart in batches .e.g. 25 at a time in a loop.